### PR TITLE
Support SCA project team assignment

### PR DESCRIPTION
**Pull request by NimrodGolan**

### CODE REVIEW

File: src/main/java/com/checkmarx/sdk/config/ScaProperties.java [hunk 1]
```diff
@@ -29,4 +29,5 @@
     private String username;
     private String password;
     private boolean enabledZipScan;
+    private String team;
```
Comment (cx-alexey-kononov) [inline src/main/java/com/checkmarx/sdk/config/ScaProperties.java]: I would call this property something like `teamForNewProjects`, which is self-documenting.

Comment (NimrodGolan) [reply src/main/java/com/checkmarx/sdk/config/ScaProperties.java]: Actually, I picked the 'team' name to be consistented with our existing 'team' property on our 'checkmarx' configuration section.

Comment (cx-alexey-kononov) [reply src/main/java/com/checkmarx/sdk/config/ScaProperties.java]: These 2 properties behave differently (as seen in com.checkmarx.sdk.service.CxService#updateProjectDetails). Giving them the same name may cause the user to make wrong assumptions and cause confusion.

Comment (NimrodGolan) [reply src/main/java/com/checkmarx/sdk/config/ScaProperties.java]: Updated.

**Review by cx-alexey-kononov — APPROVED** [review]

**Pull request merged**